sim/firesim-lib/src/main/scala/bridges/TracerVBridge.scala

```diff
@@ -154,8 +154,8 @@ class TracerVBridgeModule(key: TracerVKey)(implicit p: Parameters) extends Bridg
   lazy val toHostCPUQueueDepth  = TOKEN_QUEUE_DEPTH
   lazy val dmaSize = BigInt((BIG_TOKEN_WIDTH / 8) * TOKEN_QUEUE_DEPTH)
 
-  val uint_traces = traces map (trace => Cat(trace.valid, trace.iaddr).pad(64))
-  outgoingPCISdat.io.enq.bits := Cat(Cat(trace_cycle_counter, 
+  val uint_traces = (traces map (trace => Cat(trace.valid, trace.iaddr).pad(64))).reverse
```

Comment:
So the higher number harts were appearing first in the token?

Reply:
The higher number instruction commit ports were appearing first in the token. For ex:

Token={CycleCount...Zeros...Core0_CommitPort0, Core0_CommitPort1}.

is now

Token={CycleCount...Zeros...Core0_CommitPort1, Core0_CommitPort0}.

I personally think that the main solution to this would be to make the Chisel module understand how many cores + streams per core there are and nicely emit the proper files in the driver. However, this is a temporary solution to the ordering issue.